src/shared/variables.css

```diff
@@ -1,10 +1,11 @@
 :root {
+  font-family: 'Proxima Nova', sans-serif;
```

Comment:
Sorry I didn't realize that was a style rule in `colors.css`  - I thought it was a CSS variable.
Safe to just omit it, but it shouldn't hurt anything.

Reply:
So I can remove this from variables and colors.css and then recreate the font variable and use that for Menu.Item and Pill

Reply:
Yes, exactly. If/when we want to apply a `font-family` to a component, we should re-add it to `variables.css`

If you want to do that now, great, but it can also wait if necessary.